api.md

```diff
@@ -164,3 +171,17 @@ Methods:
 
 - <code>client.webhooks.<a href="./src/finch/resources/webhooks.py">unwrap</a>(\*args) -> object</code>
 - <code>client.webhooks.<a href="./src/finch/resources/webhooks.py">verify_signature</a>(\*args) -> None</code>
+
+# Employer
```

Comment:
Let's exclude this as per https://github.com/Finch-API/finch-api-python/pull/111/commits/a05398c6e06119d43cbe124cb2ecf92db24dbe3f#r1344054349